test/components/notification.spec.js

```diff
@@ -1,203 +1,178 @@
 
-      expect($fixture.find('.notification').length).to.equal(0)
-      expect(
-        $fixture.find('[data-notification]').data('notification')
-      ).to.be.undefined
+      expect(fixtureElement.querySelectorAll('.notification').length).to.be.equal(0)
+      expect(fixtureElement.querySelectorAll('[data-notification]').length).to.be.equal(0)
     })
   })
 
-  describe('_createNotification', () => {
+  describe('@createNotification', () => {
     it('should create notification in DOM', () => {
-      instance._createNotification()
+      instance.createNotification()
 
-      expect($fixture.find('.notification')).to.exist
+      expect(fixtureElement.querySelector('.notification')).to.be.exist
     })
 
     it('should not create notification in DOM, if dynamic is false', () => {
       options.dynamic = false
-      let newIstance = new Notification(fixture.load('notification.html')[0], options)
+      const newInstance = new Notification(fixtureElement, options)
 
-      newIstance._createNotification()
+      newInstance.createNotification()
 
-      expect(newIstance.$box === newIstance.$element).to.be.true
+      expect(newInstance.box === newInstance.element).to.be.true
     })
 
     it('should not create notification in DOM, if message is empty', () => {
       options.message = null
 
-      let newIstance = new Notification(fixture.load('notification.html')[0], options)
+      const newInstance = new Notification(fixtureElement, options)
 
-      newIstance._createNotification()
+      newInstance.createNotification()
 
-      expect($fixture.find('.notification').length).to.equal(0)
+      expect(fixtureElement.querySelectorAll('.notification').length).to.be.equal(0)
     })
 
     it('should add type class if declared', () => {
       options.type = 'warning'
 
-      let newIstance = new Notification(fixture.load('notification.html')[0], options)
+      const newInstance = new Notification(fixtureElement, options)
 
-      newIstance._createNotification()
+      newInstance.createNotification()
 
-      expect($fixture.find('.notification-warning')).to.exist
+      expect(fixtureElement.querySelector('.notification-warning')).to.be.exist
     })
   })
 
-  describe('_createCloseButton', () => {
+  describe('@createCloseButton', () => {
     it('should not create close element, if dynamic is false', sinon.test(function () {
-      let spy = this.spy($.fn, 'append')
-
       options.dynamic = false
 
-      let newIstance = new Notification(fixture.load('notification.html')[0], options)
+      const newInstance = new Notification(fixtureElement, options)
 
-      newIstance.init()
+      newInstance.init()
 
-      expect(spy.notCalled).to.be.true
+      expect(newInstance.close).to.be.null
     }))
 
     it('should create close element', () => {
       instance.init()
 
-      expect(instance.$box.find('.notification-close')).to.exist
+      expect(instance.box.querySelector('.notification-close')).to.be.exist
```

Comment:
Nãão, o `be` era só pros casos de `to.be.true`, esse pode deixar só `to.exist`

Reply:
uai ehuehuehuehue